### PR TITLE
[tests] Added tests and mysql/postgres services on travis

### LINKED ISSUE #2: [ci] Add MySQL and PostgreSQL to travis build
Configure travis-ci build to run tests both on mysql and postgresql

### PR DESCRIPTION
**Pull request by lillopaco**
Added MySQL and PostgreSQL to travis build and  we have decided to manage the parameters of the database on travis using django-environ. 
Added tests for models and admin.

Fix #2
Related to #1

### CODE REVIEW

**Comment by yakky** [comment]
See #9 for further improvements

**Comment by yakky** [comment]
@nemesisdesign Just give the review, I'll the merge myself

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 language: python
 sudo: false
+services:
+  - mysql
+  - postgresql
+
 cache: pip
 
+
```
Comment (nemesisdesign) [inline .travis.yml]: remove this extra blank line

**Comment by yakky** [comment]
@nemesisdesign ok WRT to ``runtests.py`` remarks
:100: for squashing. My original plan was to squash everything from github, and leave the concepts around rebase at a later time

File: tests/settings.py [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 DEBUG = True
 
 ALLOWED_HOSTS = []
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'freeradius.db',
-    }
+    'default': env.db(),
```
Comment (yakky) [inline tests/settings.py]: rewrite as ``'default': env.db(default='sqlite:///django-freeradius.db'),``

**Comment by yakky** [comment]
@lillopaco :100: squashing+merging

**Pull request merged**